server/src/routes/api/v1/auth/index.ts

```diff
@@ -38,4 +45,24 @@ export class AuthRouteV1 {
 
     return this._router;
   }
+
+  private checkLoginRequest (req: Request, res: Response, next: NextFunction): any {
```

Comment:
I think this should be done via input validation in any way. Maybe you need a model for the login